packages/fuel-indexer-tests/tests/e2e/indexing_postgres.rs

```diff
@@ -165,24 +165,32 @@ async fn test_can_trigger_and_index_blocks_and_transactions_postgres() {
 
     let contract = connect_to_deployed_contract().await.unwrap();
     let app = test::init_service(app(contract)).await;
+    let req = test::TestRequest::get().uri("/block_height").to_request();
```

Comment:
Why the call here?

Reply:
I found that this this test may fail if it's run after the other tests directly after other tests as we don't enforce the block heights to be unique upon insertion and the fuel node fixture's chain resets on every run, which leads to multiple records where `block_height = 1`. So it checks for a block that has the height of 1, but if you run a couple of other tests before this and then run the query, it may grab the wrong block and the TX length assertion will fail. Thus, I added a check for the current block height plus one to ensure that the correct block is selected.

If you want to verify the failure, run the `e2e::graphql_api_postgres` tests and then run the `e2e::indexing_postgres` tests directly after with no database refresh; you should see a failure in this test.